Makefile

```
@@ -398,6 +398,6 @@ project/validate:
 	[ "$$(ls -A $(PWD))" ] || (echo "Project root folder is empty. Please confirm docker has been configured with the correct permissions" && exit 1)
 	@ echo "[$@]: Target test folder validation successful"
 
-install: shellcheck/install bats/install black/install pylint/install pylint-pytest/install pydocstyle/install ec/install yamllint/install cfn-lint/install yq/install bumpversion/install
+install: black/install pylint/install pylint-pytest/install pydocstyle/install ec/install yamllint/install cfn-lint/install yq/install bumpversion/install
```

@kumoy Looks like the `bats/install` target was actually creating the directory `$HOME/bin`. We have a target to create the directory, but we were not invoking it. This change should fix the failure in travis-ci... 

```suggestion
install: $(BIN_DIR)
install: black/install pylint/install pylint-pytest/install pydocstyle/install ec/install yamllint/install cfn-lint/install yq/install bumpversion/install
```